secret/secrets_manager_client_test.go

```diff
@@ -61,11 +72,77 @@ func TestSecretsManagerCreateAndDeleteSecret(t *testing.T) {
 		require.NoError(t, err)
 		require.NotZero(t, out)
 
+		defer Cleanup(out)
+
+		require.NotZero(t, out.Name)
+		require.NotZero(t, out.ARN)
+
+	})
+
+	t.Run("GetFailsWithInvalidInput", func(t *testing.T) {
+		out, err := c.GetSecretValue(ctx, &secretsmanager.GetSecretValueInput{})
+		assert.Error(t, err)
+		assert.Zero(t, out)
+	})
+
+	t.Run("UpdateFailsWithInvalidInput", func(t *testing.T) {
+		out, err := c.UpdateSecret(ctx, &secretsmanager.UpdateSecretInput{})
+		assert.Error(t, err)
+		assert.Zero(t, out)
+	})
```

Comment:
Can you test that updating the secret fails if it doesn't exist (i.e. just use `UpdateSecret` with a random string prefixed with AWS_SECRET_PREFIX).